source/MRMesh/MRSceneLoad.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "MRObject.h"
+#include "MRResumableTask.h"
+
+namespace MR::SceneLoad
+{
+
+/// Scene loading result
+struct SceneLoadResult
+{
+    /// The loaded scene or empty object
+    std::shared_ptr<Object> scene;
+    /// Marks whether the scene was loaded from a single file or was built from scratch
+    bool isSceneConstructed = false;
```

Comment:
false means loaded?

Reply:
Yes; I'll update the comment to clarify it.